core/src/test/java/greencity/service/impl/PlaceServiceImplTest.java

```diff
@@ -160,16 +156,16 @@
         .modifiedDate(ZonedDateTime.now())
         .build();
 
-    @Before
-    public void init() {
+    @BeforeEach
+    void init() {
         MockitoAnnotations.initMocks(this);
         placeService = new PlaceServiceImpl(placeRepo, modelMapper, proposePlaceMapper, categoryService,
-            locationService, discountValueMapper, userService, openingHoursService, discountService,
+            locationService, specificationService, userService, openingHoursService, discountService,
             notificationService, zoneId, rabbitTemplate);
     }
 
     @Test
-    public void saveTest() {
+    void saveTest() {
```

Comment:
Why you remove public ?

Reply:
In junit5 test classes and methods do not have to be public anymore. Package visibility suffices. I think this is a sensible choice and in line with how we intuit the different visibility modifiers.